go.mod

```diff
@@ -1,14 +1,13 @@
 module github.com/snowplow/sql-runner
 
-go 1.13
+go 1.15
```

Comment:
@jbeemster I meant to ask about this - any issues with bumping golang version? Can revert back to 1.13 if so. :)

Reply:
No issue from me - assuming everything works still :)